### PR TITLE
LPS-176715 Create toggle to enable entry translation when creating a field

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-176715

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AddObjectField.tsx [hunk 1]
```diff
@@ -91,6 +95,11 @@ function ModalAddObjectField({
 		onSubmit,
 	});
 
+	const showEnableTranslationToggle =
+		values.businessType === 'Text' ||
+		values.businessType === 'LongText' ||
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AddObjectField.tsx:100]: @danvb112 Please, sort the condition in alphabetical order. Thanks :)

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:object

**Comment by naraandrade** [comment]
@danvb112 the failures don't seem related to your changes, but do a double check, please, before sending it directly to Brian.

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**